### ISSUE TITLE
error on shellscript and spykingcircus

### ISSUE THREAD

**Issue by ferchaure** [SpikeInterface/spikesorters#164]
Hi I tested spyking-circus using the current commit and I got the following odd errors. Any ideas? 

(exactly same code using the spikeinterface version installed using pip works fine)

```
RUNNING SHELL SCRIPT: F:\GitHub\spikeslink_tests\spykingcircus_output\run_spykingcircus.bat
Traceback (most recent call last):
  File "F:\GitHub\spikesorters\spikesorters\basesorter.py", line 159, in run
    self._run(recording, self.output_folders[i])
  File "F:\GitHub\spikesorters\spikesorters\spyking_circus\spyking_circus.py", line 145, in _run
    shell_script.start()
  File "F:\GitHub\spikesorters\spikesorters\utils\shellscript.py", line 83, in start
    for line in self._process.stdout:
  File "E:\programs\Anaconda3\envs\spikes_link\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x8f in position 8: character maps to <undefined>
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "E:\programs\Anaconda3\envs\spikes_link\lib\site-packages\IPython\core\interactiveshell.py", line 3331, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-2-cca46161feda>", line 1, in <module>
    runfile('F:/GitHub/spikeslink_tests/run_spike_circus_blocks_si.py', wdir='F:/GitHub/spikeslink_tests')
  File "E:\programs\PyCharm 2019.3\plugins\python\helpers\pydev\_pydev_bundle\pydev_umd.py", line 197, in runfile
    pydev_imports.execfile(filename, global_vars, local_vars)  # execute the script
  File "E:\programs\PyCharm 2019.3\plugins\python\helpers\pydev\_pydev_imps\_pydev_execfile.py", line 18, in execfile
    exec(compile(contents+"\n", file, 'exec'), glob, loc)
  File "F:/GitHub/spikeslink_tests/run_spike_circus_blocks_si.py", line 35, in <module>
    sorting = so.run_spykingcircus(recording,**w_par)
  File "F:\GitHub\spikesorters\spikesorters\sorterlist.py", line 483, in run_spykingcircus
    return run_sorter('spykingcircus', *args, **kwargs)
  File "F:\GitHub\spikesorters\spikesorters\sorterlist.py", line 86, in run_sorter
    sorter.run(raise_error=raise_error, parallel=parallel, n_jobs=n_jobs, joblib_backend=joblib_backend)
  File "F:\GitHub\spikesorters\spikesorters\basesorter.py", line 170, in run
    raise SpikeSortingError(f"Spike sorting failed: {err}. You can inspect the runtime trace in "
spikesorters.sorter_tools.SpikeSortingError: Spike sorting failed: 'charmap' codec can't decode byte 0x8f in position 8: character maps to <undefined>. You can inspect the runtime trace in the spykingcircus.log of the output folder.'

```

**Comment by alejoe91** [comment]
HI @ferchaure 

Sorry I missed this! Maybe @yger can help :)

**Comment by yger** [comment]
Not sure I can do much.... Could it be that you have weird character in a path, somewhere?

**Comment by ferchaure** [comment]
I guess that the doesn't make sense to go back to the old commit. I  just checked again with the current pip versions of spikeinterface and spykingcircus and I got the same error. 

I run just the created bat file `run_spykingcircus.bat `and that works fine. I a windows thing, I tried exactly the same on Linux without any problem.
I think the issue is related to the ShellScript, some odd character in the output of spykingcircus that ShellScript can't handle
